src/extension-test/unit/cursor-doc/paredit-test.ts

```diff
@@ -177,8 +177,7 @@ describe('paredit', () => {
         const b = docFromTextNotation('(a| b (c\r\n d)| e)');
         const [start, end] = textAndSelection(b)[1];
         const actual = paredit.forwardHybridSexpRange(a);
-        // off by 1 because \r\n is treated as 1 char?
-        expect(actual).toEqual([start, end - 1]);
```

Comment:
🙏

Reply:
Thanks for confurming this was not intented :)